app/views/layouts/application.html.erb

```diff
@@ -6,30 +6,57 @@
 
     <title><%= content_for?(:title) ? yield(:title) : "Caesar" %></title>
 
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.6/css/bootstrap.min.css"
```

Comment:
How do we feel (not taking a position / being dogmatic) about importing scripts and styles from a CDN? We don't do this in PFE. Also noticing that this is an alpha of Bootstrap 4, any sense of how stable this API is going to be?